docs/getting-started/upgrade-to-v4.md

```diff
@@ -5,6 +5,26 @@ title: Upgrade Guide (v4)
 
 NextAuth.js version 4 included a few breaking changes from the last major version (3.x). So we're here to help you upgrade your applications as smoothly as possible. It should be possible to upgrade from any version of 3.x to the latest 4 release by following the next few migration steps.
 
+:::note
+Version 4 is currently in beta. We encourage users to try it out as we don't plan to change the API any more, but be aware that if a bug-fix requires so, we will do that without further notice.
+:::
+
+You can upgrade to the new version by running:
+
+```sh
+npm i next-auth@beta
+```
+
+#### Verify the correct version
```

Comment:
I added this as a header, so we can link to it. I expect many people will have this issue. :confused: